README.md

```diff
@@ -1,6 +1,58 @@
 # covid19-india
 
+## Repo Structure
+    The repository is structured as follows:
+    ./
+    |-- config    : Various config params to be entered by humans go here
+    |-- data      : All data go in here, files are tracked by git-lfs
+    |-- outputs   : All output data go in here, files are tracked by git-lfs
+    |-- docs      : All documentation other than README goes here
+    |-- Makefile  : Makefile to clean data, run tests etc
+    |-- notebooks : All notebooks go here
+    |-- setup.sh  : Setup script to setup path etc
+    `-- src       : All production code, resusable code components go here
+        |-- configs        : All configs params go here
```

Comment:
To avoid confusion with "configs" on line 6, would you like to rename it config_defs  and "All config param definitions go here"